activerecord/test/models/review.rb

```diff
@@ -0,0 +1,3 @@
+class Review < ActiveRecord::Base
```

Comment:
Is there an existing polymorphic association we can use, instead of adding a new one? There are a lot of models, so it's generally better to avoid adding more.